methods/get_meta.nf

```diff
@@ -1,14 +1,45 @@
+idList = []     // Global variable to save IDs from samplesheet
+
 // Function for platform and paired-end or single-end
 def get_meta(LinkedHashMap row) {
-    platforms = ["illumina", "nanopore", "pacbio", "iontorrent"]
-    if (row.platform in platforms) {
-        if (row.read2) {
-        meta = tuple(row.id, tuple(file(row.read1), file(row.read2)), row.platform)
-        } else {
-        meta = tuple(row.id, tuple(file(row.read1)), row.platform)
-        }
+  platforms = ["illumina", "nanopore", "pacbio", "iontorrent"]
+
+  // Error messages
+  idErrorMessage = "\nERROR: Please check input samplesheet -> ID '${row.id}' is not a unique name in the samplesheet."
```

Comment:
I think this is a good effort but I reckon that we should maybe consider making a new method called `check_input.nf` because this check plays no role in the act of getting the metadata. Revert this method to it's original format and create a method file.